api-reference/10 UI Components/dxDraggable/1 Configuration/onDragEnd.md

```diff
@@ -23,13 +23,13 @@ The UI component that raised the event.
 #include common-ref-eventparam
 
```

Comment:
Please update the following line above:

Line 8:
`A function that is called when drag gesture is finished.`

`A function that is called when a drag gesture is finished.`